quic/s2n-quic-qns/Cargo.toml

```diff
@@ -26,6 +26,7 @@ openssl-sys = { version = "0.9", features = ["vendored"] }
 rustls = { version = "0.21", features = ["dangerous_configuration", "quic"] }
 s2n-quic-core = { path = "../s2n-quic-core", features = ["testing"] }
 s2n-quic-h3 = { path = "../s2n-quic-h3" }
+s2n-tls = { version = "=0.0.36", features = ["quic"] }
```

Comment:
This should be exported from `s2n_quic::provider::tls::s2n_tls`

https://github.com/aws/s2n-quic/blob/047f695aeb6219fc2bde54b95cd1915d73d1c92b/quic/s2n-quic/src/provider/tls.rs#L277